lib/_emerge/actions.py

```diff
@@ -1648,7 +1651,8 @@ def create_cleanlist():
 
                 if debug:
                     writemsg_level(
-                        "Candidates: [%s]\n" % ", ".join("'%s'" % (x,) for x in atoms),
+                        "Candidates: [%s]\n"
+                        % ", ".join("'{}'".format(x) for x in atoms),
```

Comment:
Looks like it missed a `.format` opportunity here.

Reply:
Hm, yeah, seems like there's some moderately odd constructs that it doesn't handle:

```
 ~ $ cat pyupgrade-test.py
msg = [
    "%s" % "foo"
]

msg = ["{} {}".format("foo", "%s" % "bar")]
 ~ $ pyupgrade pyupgrade-test.py
 ~ $ cat pyupgrade-test.py
msg = [
    "%s" % "foo"
]

msg = ["{} {}".format("foo", "%s" % "bar")]
```